langchain/package.json

```diff
@@ -887,6 +891,9 @@
     "@aws-sdk/client-sagemaker-runtime": "^3.310.0",
     "@aws-sdk/client-sfn": "^3.310.0",
     "@aws-sdk/credential-provider-node": "^3.388.0",
+    "@aws-sdk/protocol-http": "^3.374.0",
```

Comment:
I think these AWS deps should be removed? Maybe a bad merge?

Reply:
Yeah, looks like a bad merge. I will remove that